core/src/test/java/org/elasticsearch/cluster/SpecificMasterNodesIT.java

```diff
@@ -111,16 +112,18 @@ public void testCustomDefaultMapping() throws Exception {
         internalCluster().startNode(settingsBuilder().put(Node.NODE_DATA_SETTING.getKey(), true).put(Node.NODE_MASTER_SETTING.getKey(), false));
 
         createIndex("test");
-        assertAcked(client().admin().indices().preparePutMapping("test").setType("_default_").setSource("_timestamp", "enabled=true"));
+        assertAcked(client().admin().indices().preparePutMapping("test").setType("_default_").setSource("timestamp", "type=date"));
 
         MappingMetaData defaultMapping = client().admin().cluster().prepareState().get().getState().getMetaData().getIndices().get("test").getMappings().get("_default_");
-        assertThat(defaultMapping.getSourceAsMap().get("_timestamp"), notNullValue());
+        Map<?,?> properties = (Map<?, ?>) defaultMapping.getSourceAsMap().get("properties");
```

Comment:
I like the wildcard better since it does not make any assumption about the data that this map stores.